pkg/models/dashboard_version.go

```diff
@@ -15,11 +15,12 @@ var (
 // A DashboardVersion represents the comparable data in a dashboard, allowing
 // diffs of the dashboard to be performed.
 type DashboardVersion struct {
-	Id            int64 `json:"id"`
-	DashboardId   int64 `json:"dashboardId"`
-	ParentVersion int   `json:"parentVersion"`
-	RestoredFrom  int   `json:"restoredFrom"`
-	Version       int   `json:"version"`
+	Id            int64  `json:"id"`
+	DashboardId   int64  `json:"dashboardId"`
+	UID           string `json:"uid"`
```

Comment:
The `TestOrgUsersAPIEndpoint_LegacyAccessControl_FolderAdmin` fails with:
`table dashboard_version has no column named u_i_d`
because of this change.
This is a model and when is passed [here](https://github.com/grafana/grafana/blob/ee8e1251340688bc98e0d2f580e8ce1ce456ff29/pkg/services/dashboards/database/database.go#L483) the ORM expects that there is such a column in the respective table `dashboard_version`
```suggestion
```

You don't actually need this change. You can just modify this line:
https://github.com/grafana/grafana/blob/431444af332bb58cd114547e681c0368e0824659/pkg/api/dashboard.go#L605
to set the dashboard UID from the `dashUID` variable.